.teamcity/common/Os.kt

```diff
@@ -16,6 +16,10 @@
 
 package common
 
-enum class Os(val agentRequirement: String, val ignoredSubprojects: List<String> = emptyList()) {
-    linux("Linux"), windows("Windows"), macos("Mac", listOf("integ-test", "native", "plugins", "resources", "scala", "workers", "wrapper", "platform-play", "tooling-native"))
+enum class Os(val agentRequirement: String, val ignoredSubprojects: List<String> = emptyList(), val androidHome: String = "/opt/android/sdk") {
```

Comment:
I think it would be better to write `/opt/android/sdk` out instead of relying on this default.

Reply:
Done.